crates/config/src/utils.rs

```diff
@@ -259,6 +258,39 @@ where
     Ok(num)
 }
 
+/// Helper type to parse both `u64` and `U256`
+#[derive(Copy, Clone, Deserialize)]
+#[serde(untagged)]
+pub enum Numeric {
+    /// A [U256] value.
+    U256(U256),
+    /// A `u64` value.
+    Num(u64),
+}
+
+impl From<Numeric> for U256 {
+    fn from(n: Numeric) -> U256 {
+        match n {
+            Numeric::U256(n) => n,
+            Numeric::Num(n) => U256::from(n),
+        }
+    }
+}
+
+impl FromStr for Numeric {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        if let Ok(val) = s.parse::<u128>() {
+            Ok(Numeric::U256(U256::from(val)))
```

Comment:
This isn't really right, right? As you'll fail for >u128. https://docs.rs/serde_json/latest/serde_json/value/struct.Number.html